Source/Scene/OrthographicFrustum.js

```diff
@@ -312,6 +317,51 @@ define([
     };
 
     /**
+     * Returns the pixel's width and height in meters.
+     *
+     * @param {Number} drawingBufferWidth The width of the drawing buffer.
+     * @param {Number} drawingBufferHeight The height of the drawing buffer.
+     * @param {Cartesian2} result The object onto which to store the result.
+     * @param {Number} [distance=near plane distance] The distance to the near plane in meters.
```

Comment:
`distance` is not optional, so it should just be `distance`, not `[distance=near plane distance]`